overlord/fdestate/fdemgr.go

```diff
@@ -38,6 +41,10 @@ func Manager(st *state.State, runner *state.TaskRunner) *FDEManager {
 		state: st,
 	}
 
+	boot.InstallResealKeyForBootChains(func(method device.SealingMethod, rootdir string, params *boot.ResealKeyForBootChainsParams, expectReseal bool) error {
+		return m.resealKeyForBootChains(method, rootdir, params, expectReseal)
+	})
```

Comment:
this should work too
```suggestion
	boot.InstallResealKeyForBootChains(m.resealKeyForBootChains)
```

Reply:
Done along with related comment from @pedronis